CodeiumVS/CodeiumVS.csproj

```diff
@@ -125,6 +126,9 @@
     <PackageReference Include="Microsoft.VisualStudio.Language">
       <Version>17.8.222</Version>
     </PackageReference>
+    <PackageReference Include="Microsoft.VisualStudio.SDK.VsixSuppression">
+      <Version>14.1.37</Version>
+    </PackageReference>
```

Comment:
Sometimes when debugging, it seems to corrupt the MEF component cache, this was added to fix it https://www.cazzulino.com/nuking-component-model-cache.html

Reply:
Good find